index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css.css">
+
```

Comment:
Add meta tag that will describe your project 
like example:
`<meta name="description" content="This is meta description Sample. We can add up to 158.">`